runtime/binding-grpc-kafka/src/main/java/io/aklivity/zilla/runtime/binding/grpc/kafka/internal/config/GrpcKafkaWithProduceResult.java

```diff
@@ -94,7 +94,11 @@ public class GrpcKafkaWithProduceResult
         hash.updateHash(correlation.method.value());
         hash.updateHash(method.value());
         hash.updateHash(correlation.replyTo.value());
```

Comment:
Does this also belong in the condition below to verify replyTo is non-null?
Seems like there should be symmetry with builder.item(...) calls further below in `headers(...)` method.